scripts/generateAssetData/avalanche/index.ts

```diff
@@ -18,7 +18,7 @@ export const getAssets = async (): Promise<Asset[]> => {
     return []
   })
 
-  const allAssets = uniqBy(assets.concat(portalsAssets).concat([avax]), 'assetId')
+  const allAssets = uniqBy(portalsAssets.concat(assets).concat([avax]), 'assetId')
```

Comment:
Note for this guy and all occurences: we `uniqBy` the assets to remove dupes, so the first will be used.
Since Coingecko comes first, and *does* contain some LP tokens as well, we need to have the Portal ones first in the array so that `isPool` is present, and so that this PR actually works.

Reply:
Actually revert me - intent is good but there's a better way. This will rug some of our current assets e.g 

<img width="314" alt="image" src="https://github.com/user-attachments/assets/2d6ee1fa-da41-4212-bd56-41ad05aef1ec">
<img width="346" alt="image" src="https://github.com/user-attachments/assets/4d5f6fdd-8e20-4dca-a90e-2e6144e4e2b1">

Rather, this should sort by `isPool` assets first, *then* do a `uniqBy`

Reply:
500c9e80f3c640b7d6d8ce217fb56d572e6b71ad - keeping open for reference